src/ui/public/index_patterns/__tests__/_index_pattern.js

```diff
@@ -12,8 +12,11 @@ import IndexPatternsMapperProvider from 'ui/index_patterns/_mapper';
 import UtilsMappingSetupProvider from 'ui/utils/mapping_setup';
```

Comment:
Could you explain the reason for the changes in this file? I don't understand how they connect to the agg changes.